inference-engine/tests/functional/plugin/shared/src/single_layer_tests/non_max_suppression.cpp

```diff
@@ -79,12 +79,22 @@ void NmsLayerTest::Compare(const std::vector<std::vector<std::uint8_t>> &expecte
         const auto &precision = actual->getTensorDesc().getPrecision();
         size_t size = expected.size() / actual->getTensorDesc().getPrecision().size();
         switch (precision) {
-            case Precision::FP32:
+            case Precision::FP32: {
                 LayerTestsCommon::Compare(reinterpret_cast<const float *>(expectedBuffer), reinterpret_cast<const float *>(actualBuffer), size, threshold);
+                const auto fBuffer = lockedMemory.as<const float *>();
+                for (int i = size; i < actual->size(); i++) {
+                    ASSERT_TRUE(fBuffer[i] == -1.f) << "Invalid default value at index " << i;
```

Comment:
Could you change the error message a little?
```suggestion
                    ASSERT_TRUE(fBuffer[i] == -1.f) << "Invalid default value: " << fBuffer[i] << " at index: " << i;
```

Reply:
done